CHANGELOG.md

```diff
@@ -1,11 +1,20 @@
+## 10.0.0-BETA.9 (2020-XX-XX)
+
+### Enhancements
+* Users can now opt out from allowing queries to be launched from the UI thread by using `RealmConfiguration.Builder.allowQueriesOnUiThread(false)`. Doing so will result in a `RealmException` when calling `RealmQuery.findAll()` or `RealmQuery.findFirst()`. Queries will still be allowed from any thread by default, although we recommend using a non-UI thread to launch queries, or, alternatively, using `RealmQuery.findAllAsync()` or `RealmQuery.findFirstAsync()`
```

Comment:
We should also mention `RealmQuery.<count/min/max/average>` and missing `.` at the end.